### ISSUE TITLE
Updater using malformed host url

### ISSUE THREAD

**Issue by patricktokeeffe** [owncloud/news#820]
Using 5.2.8, latest release compatible with owncloud 8.0 (stable), and owncloud 8.0.3:

When I install and try to run the custom updater script, it fails repeatedly with the following message. Adding a trailing slash to the url in my config file didn't resolve it.

````
root@example:/var/www/owncloud/apps/news/bin/updater# owncloud-news-updater -c /etc/owncloud/news/updater.ini
2015-07-03 16:04:10,907 - ownCloud News Updater - ERROR - http://localhost/oc/index.php/apps/news/api/v1-2: HTTPConnectionPool(host='example.netoc', port=80): Max retries exceeded with url: /index.php/apps/news/api/v1-2/cleanup/before-update (Caused by <class 'socket.gaierror'>: [Errno -2] Name or service not known) Trying again in 30 seconds
````

The host argument passed to `HTTPConnectionPool` is invalid. I haven't figured out where localhost becomes my external hostname (edited to "example" above). Tracebacks shown after sending a keyboard interrupt direct me to [this line][1], where `self.before_cleanup_url` should have the value `http://localhost/oc/index.php/apps/news/api/v1-2/cleanup/before-update`.

  [1]: https://github.com/owncloud/news/blob/5.2.8/bin/updater/owncloud_news_updater/updater.py#L52

**Comment by Raydiation** [comment]
Please post your config and try an IP instead of local host, very likely a setup problem

**Comment by Raydiation** [comment]
Closing because of inactivity

**Comment by patricktokeeffe** [comment]
Just to follow-up: I gave up on the custom updater after encountering the idea of a cron entry to delete stale cron lockfiles.

**Comment by Raydiation** [comment]
I've created a patch for core BTW, the issue should be fixed in 8.1
